api/MediaPositionState.json

```diff
@@ -0,0 +1,248 @@
+{
+  "api": {
+    "MediaPositionState": {
```

Comment:
OK, this steps into a complex question about whether and how to represent IDL dictionaries in BCD. In this case, since, as far as I can tell, `MediaPositionState` only matters for one method, we should move the subfeatures of `MediaPositionState` to `api.MediaSession.setPositionState` (e.g., described as `"<code>state.duration</code> argument"`) and get rid of this file.

Now, _why_ we should do this pretty complex. I'm sorry about this. If you're really curious, you can read #6002 and #6265, but it's a long story. Here's my highly-opinionated summary:

The Media Session spec names several abstract dictionaries as a convenience to spec authors and browser developers. However, these dictionary names don't have a concrete manifestation for web developers; they indirectly appear as patterns for concrete return values and method arguments. Consequently, it's an open question what it means to "support" an abstract dictionary.

That said, we don't have a good answer in BCD or on MDN for how to represent these things yet. There's way more discussion on this point in #7287. In the past (e.g., #6265), I've compromised when such dictionaries have touched many APIs, but I've resisted introducing them in cases like this one.

As far as I can tell, `MediaPositionState` represents one thing only: the structure of the object argument to `MediaSession.setPositionState()`. I don't know what it means to support `MediaPositionState`, but I do have a sense for what it means for `setPositionState()` to support `state.duration`. We can represent this information as subfeatures of the method and call it a day.